regimes/my/tax_categories.go

```diff
@@ -0,0 +1,111 @@
+package my
+
+import (
+	"github.com/invopop/gobl/cal"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/num"
+	"github.com/invopop/gobl/tax"
+)
+
+var taxCategories = []*tax.CategoryDef{
+	//
+	// Sales Tax (part of SST)
+	//
+	{
+		Code: "sales-tax", // Custom subcategory under SST
```

Comment:
The convention in GOBL is that Codes are always short upper-case characters, normally representing a common name. In this case, I'd used `SST` from a constant defined in the `tax` package.